Cartfile

```diff
@@ -2,6 +2,7 @@ binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.7
 github "mapbox/MapboxDirections.swift" ~> 0.15
 github "mapbox/turf-swift" ~> 0.0.3
 github "rs/SDWebImage" ~> 4.1
-github "frederoni/aws-sdk-ios" ~> 2.6
 github "mapbox/mapbox-telemetry-ios" ~> 0.2
 github "ceeK/Solar" ~> 2.1.0
+github "mapbox/mapbox-voice-swift" "master"
+github "frederoni/aws-sdk-ios" ~> 2.6
```

Comment:
Will remove once this PR is ready. It makes it easier for me to keep both while developing between branches.